### ISSUE TITLE
install fails looking for version.php in incorrect path

### ISSUE THREAD

**Issue by durzo** [moodlehq/moodle-plugin-ci#138]
running inside a docker container, whether i specify `--moodle` or not, after successfull install it is looking for a version.php in the current dir, instead of where moodle was actually installed.

```
root@dd34ae176c22:/var/www/html# moodle-plugin-ci install --branch="MOODLE_311_STABLE" --db-host=172.17.0.2 --db-name="behat_test" --db-pass="password" --no-init -vvv
[info] Starting install
[info] Cloning Moodle
  RUN  git clone --depth=1 --branch MOODLE_311_STABLE git://github.com/moodle/moodle.git moodle
  ERR  Cloning into 'moodle'...
Updating files: 100% (22204/22204), done.
  ERR  
  RES  Command ran successfully
[info] Moodle assets
[debug] Creating Moodle data directories
[debug] Create Moodle database
  RUN  env PGPASSWORD='password' psql -c 'CREATE DATABASE "behat_test";' -U 'postgres' -h '172.17.0.2'
  OUT  CREATE DATABASE
  OUT  
  RES  Command ran successfully
[debug] Creating Moodle's config file
[info] Install plugins

In CodeParser.php line 36:
                                                    
  [InvalidArgumentException]                        
  Failed to find '/var/www/html/version.php' file.  
                                                    

Exception trace:
  at /var/www/html/ci/src/Parser/CodeParser.php:36
 MoodlePluginCI\Parser\CodeParser->loadFile() at /var/www/html/ci/src/Parser/CodeParser.php:57
 MoodlePluginCI\Parser\CodeParser->parseFile() at /var/www/html/ci/src/Bridge/MoodlePlugin.php:89
 MoodlePluginCI\Bridge\MoodlePlugin->getComponent() at /var/www/html/ci/src/Bridge/MoodlePluginCollection.php:52
 MoodlePluginCI\Bridge\MoodlePluginCollection->sortByDependencies() at /var/www/html/ci/src/Installer/PluginInstaller.php:67
 MoodlePluginCI\Installer\PluginInstaller->install() at /var/www/html/ci/src/Installer/Install.php:40
 MoodlePluginCI\Installer\Install->runInstallation() at /var/www/html/ci/src/Command/InstallCommand.php:117
 MoodlePluginCI\Command\InstallCommand->execute() at /var/www/html/ci/vendor/symfony/console/Command/Command.php:255
 Symfony\Component\Console\Command\Command->run() at /var/www/html/ci/vendor/symfony/console/Application.php:992
 Symfony\Component\Console\Application->doRunCommand() at /var/www/html/ci/vendor/symfony/console/Application.php:255
 Symfony\Component\Console\Application->doRun() at /var/www/html/ci/vendor/symfony/console/Application.php:148
 Symfony\Component\Console\Application->run() at /var/www/html/ci/bin/moodle-plugin-ci:83

install [--moodle MOODLE] [--data DATA] [--repo REPO] [--branch BRANCH] [--plugin PLUGIN] [--db-type DB-TYPE] [--db-user DB-USER] [--db-pass DB-PASS] [--db-name DB-NAME] [--db-host DB-HOST] [--not-paths NOT-PATHS] [--not-names NOT-NAMES] [--extra-plugins EXTRA-PLUGINS] [--no-init] [--node-version NODE-VERSION]
```

**Comment by durzo** [comment]
some more info:
This is inside a local docker container, testing things before pushing to gitlab-ci.
I tried setting the CI_PROJECT_DIR and TRAVIS_BUILD_DIR with no luck.
I ended up pushing to gitlab and it fails in the same way on gitlab-ci.

My repo is empty, consisting only of the CI file, the end goal is to have it run behat against other repositories that will be cloned into the container - not code inside this repo.
